Classes/RequestHandler/AbstractRecordRequestHandler.php

```diff
@@ -121,7 +122,7 @@ protected function compileData(): void
      */
     public function handle(): ResponseInterface
     {
-        if (empty($this->data)) {
+        if ($this->data === []) {
```

Comment:
Why not use is_array?

Reply:
I'm checking for an empty array, not a non-array. `$this->data` is typed as array initiated as `[]`, so it will always be an array.